backend/src/main/java/de/ffmjava/capstone/backend/user/CustomApiErrorHandler.java

```diff
@@ -5,6 +5,8 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.mongodb.lang.Nullable;
```

Comment:
Maybe class Nullable should come from org.springframework.lang.Nullable ?? and this what cause build error?

Reply:
Yep, I threw it out